src/alt/index.js

```diff
@@ -77,6 +42,18 @@ class Alt {
     return makeAction(this, 'global', name, implementation, obj)
   }
 
+  registerStore(...args) {
```

Comment:
What do you think of just having `#register()` and inferring the type with `instanceof` or whatever?